modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -4,18 +4,29 @@ module AppealsApi::V1
   module DecisionReviews
     module NoticeOfDisagreements
       class EvidenceSubmissionsController < AppealsApi::ApplicationController
+        include SentryLogging
+
+        class EvidenceSubmissionRequestValidatorError < StandardError; end
```

Comment:
does `log_error_to_sentry` require an error class? This feels superfluous to me.